src/utils/actions.js

```diff
@@ -347,7 +349,7 @@ const actions = [
         }
       },
     requiredInput: ['automation'],
-    featureFlag: 'ACTION_BUTTON_AUTOMATIONS',
+    hide: (shouldHide) => shouldHide?.('ACTION_BUTTON_AUTOMATIONS'),
```

Comment:
A possible fix to my other comment is to call `hide` with all three methods.

Ex: `hide: ({ isEnabled }) => !isEnabled('ACTION_BUTTON_AUTOMATIONS')`

Reply:
Fix.